shipment_planning_maxoptra/wizard/shipment_maxoptra_schedule_import.py

```diff
@@ -70,6 +72,14 @@ def default_get(self, fields_list):
             res["shipment_planning_id"] = shipment_planning_id
         return res
 
+    @api.model
+    def _default_start_time(self):
+        user_tz = pytz.timezone(self.env.context.get('tz') or 'UTC')
+        date = pytz.utc.localize(fields.Datetime.now()).astimezone(user_tz)
+        date = date.replace(hour=6, minute=0, second=0)
+        date = date.astimezone(pytz.utc).replace(tzinfo=None)
+        return date
```

Comment:
This should work even if `now` is close to midnight (so it automatically changes day if needed):
```suggestion
    @api.model
    def _default_start_time(self):
        """Returns 6:00 of current day, regardless of user's timezone

        Returns a naive datetime object which, converted to the user's timezone,
        always shows 6:00 AM of current date
        """
        user_tz_str = self.env.user.tz or self._context.get("tz") or "UTC"
        user_tz = pytz.timezone(user_tz_str)
        now_tz = datetime.now(user_tz)
        date_tz = now_tz.replace(hour=6, minute=0, second=0, microsecond=0)
        date_utc = date_tz.astimezone(pytz.utc)
        date_naive = date_utc.replace(tzinfo=None)
        return date_naive
```
Can be refactored, I left it more lengthy for better readability :smile: